docfiles/pxtweb/cookieCompliance.ts

```diff
@@ -240,6 +248,14 @@ namespace pxt {
         }
     }
 
+    /**
+     * Checks for winrt and electron
+     */
+    function isNativeApp(): boolean {
+        return typeof Windows !== "undefined" ||
```

Comment:
do we already have helper in BrowserUtil to detect that we are in WinRT or Electron?

Reply:
There is, see electron.isElectron || pxt.winrt.isWinRT();

Reply:
I can't use the existing ones because the pxtweb stuff doesn't have any dependencies (it's embedded in every page). The electron check we use elsewhere is no good anyways because it is just checking the URL for `electron=1`